### DISCUSSION TITLE
Не работает на ТВ приставке Xiaomi TV Box S 2gen. На других Андроид устройствах работает

### DISCUSSION THREAD

**Discussion by romarus186** [dovecoteescapee/ByeDPIAndroid#127] [General]
Помогите настроить для Xiaomi TV Box S 2gen
У меня на второй ТВ приставке Tanix W2 с прошивкой Slim Box работает сразу по дефолту. 
Смартфон тоже работает на дефолте. 

Я пробовал все способы desync method. По умолчанию disorder, split position: 1. Пробовал 2. Не сработало. Потом ставил другие способы десинка. Сплит позиция на 1 не трогал. Не работает. Приложение Ютуба просто серый экран пустой. Если долго ждать то выйдет сообщение типа нету подключения.

**Comment by kvalera** [comment]
romarus186, удали ByeDPIAndroid с приставки и поставь DPITunnel-android, но нужны root права.
https://github.com/nomoresat/DPITunnel-android/releases/tag/v1.2

**Comment by kvalera** [comment]
У меня Beelink GT1 mini-2, на ней byedpi после настройки не дал никакого эффекта, а dpitunnel даже не настраивал, просто включил и YouTube летает. Попробуй настроить, может прокатит.
Запускаем приложение и предоставляем root доступ.
Переходим во вкладку 'профили'
Удаляем все профили и создаём новый нажав +
Ищем кнопку 'подобрать настройки' и жмём её.
Вводим ссылку m.youtube.com и жмём ок
После настройки у вас появятся новые параметры.
Включаем неверный SEQ для поддельных пакетов.
Включаем автоматический подбор TTL
Включаем разбивать ClientHello по середине SNI
Позиция для разбиения запроса лучше ставить цифру 6 и больше.
Нажимаем сохранить кнопкой справа снизу.

**Reply by romarus186** [reply]
> У меня Beelink GT1 mini-2, на ней byedpi после настройки не дал никакого эффекта, а dpitunnel даже не настраивал, просто включил и YouTube летает. Попробуй настроить, может прокатит. Запускаем приложение и предоставляем root доступ. Переходим во вкладку 'профили' Удаляем все профили и создаём новый нажав + Ищем кнопку 'подобрать настройки' и жмём её. Вводим ссылку m.youtube.com и жмём ок После настройки у вас появятся новые параметры. Включаем неверный SEQ для поддельных пакетов. Включаем автоматический подбор TTL Включаем разбивать ClientHello по середине SNI Позиция для разбиения запроса лучше ставить цифру 6 и больше. Нажимаем сохранить кнопкой справа снизу.

Я извиняюсь, не успел удалить коммент. Просто первый раз ютуб долго прогружался я подумал что. Тоже не работает.  Но заработало! После первого раза нормально видосы довол но быстро запускаются. В 4к смотрю все ок. Ничего не менял в настройках dpitunnel в е на дефолте заработало. Капец пол дня парился. На 4пда мне там мозг вынесли в форуме по моему тв боксу. Магиск зачем то установил модули какие-то в еем. Хз зачем. Мне просто нужен был рут и работающий ют. Кароче спасибо тебе бро! Очень помог! За свободу информации!

**Reply by dtpitop** [reply]
У меня ошибка - id профиля не может быть пустым или default. Не можешь подсказать что делать? Я упорно не понимаю

**Comment by kvalera** [comment]
Да, я всё перепробовал, эта программа единственная на приставках работает, можешь добавить её значок в шторку и появится возможность включать и выключать просто нажав на значок в шторке. Отключай её перед запуском online кинотеатров, иначе ничего не будет грузиться.

**Comment by kvalera** [comment]
И ешё, пред выключением приставки тоже отключай. Самое главное, если забудешь отключить её и удалишь, то интернета в приставке не будет, пока заново не поставишь и включишь, отключишь, тогда инет появится.

**Reply by romarus186** [reply]
Ок. Но само приложение ютуба прям грузится долго когда запускаешь, аж минуту где-то. Потом первое видео когда запускаешь тоже долго. А потом все быстро. Это после выхода из программы такое. У вас также? Какие настройки могут на это повлиять. Я выше видел вы настройки описали. Ну это методом пробы надо кароче пробовать долгая история. Завтра уже попробую. Н асегодня надоело уже париться. Пока работает и ладно.

**Comment by kvalera** [comment]
У меня быстро запускается YouTube при включенной проге. Попробуй отключить программу и сначала запусти YouTube, пусть загрузятся картинка с роликами, а затем включи программу. Если не запустится видео, то на отключенной проге включи YouTube, подожди когда загрузится, а потом сверни его с экрана и запусти программу, а потом разверни УouTube и попробуй запуситьть какой нибудь ролик.

**Comment by kvalera** [comment]
На твоей приставке установлен SmartTube (ex. SmartTubeNext) или YouTube ReVanced. Если SmartTube, то в настройках обновись до последней версии (я обновлял SmartTube), а YouTube ReVanced нужно удалить и после перезагоузки устройства поставить последний релиз.

**Reply by romarus186** [reply]
Ни то ни другое:) стоит стандартное приложение ютуб. Но я понял. Попробую их. Может в этом дело. Спасибо. В любом случае именно dpi tunnel заработал.
